subprojects/gradle/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/BuildMetricsPlugin.kt

```diff
@@ -34,6 +36,9 @@ open class BuildMetricsPlugin : Plugin<Project> {
             SentryConsumer(project),
             buildTraceConsumer(project)
         ))
+        project.tasks.register<CollectTeamcityMetricsTask>("collectTeamcityMetrics") {
+            buildId.set(project.getMandatoryStringProperty("avito.build.metrics.teamcityBuildId"))
```

Comment:
oops, it's not a real property btw
it will be resolved in task configuration, we should work on naming here
it's kinda ok, because task will not be configured in other builds than special one for collection metrics